lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -230,9 +240,19 @@ def __configure_port_routing(self, ajs):
         k8s_service_obj = service_object_dict(self.runner_params, k8s_job_name, self.__get_k8s_service_spec(ajs))
 
         k8s_ingress_obj = ingress_object_dict(self.runner_params, k8s_job_name, self.__get_k8s_ingress_spec(ajs))
+        # pretty print the objects for debugging
+        log.debug(f"Kubernetes service object: {json.dumps(k8s_service_obj, indent=4)}")
+        log.debug(f"Kubernetes ingress object: {json.dumps(k8s_ingress_obj, indent=4)}")
```

Comment:
```suggestion
```
I wonder whether this will spam logs in more high-volume environments? Would looking at the created ingress object in k8s suffice, or do you think seeing the ingress spec will be useful?

Reply:
I agree with the sentiment, but I'm not sure that most instances are on info. I know of some prod instances that run on debug, possibly to make it easier to identify tool errors and the like. Galaxy does have a trace level, so maybe we could do `log.trace` for these messages instead?

Reply:
Trace sounds good for this.

Reply:
done now, so will resolve.